pkg/kapis/devops/v1alpha3/pipelinerun/handler.go

```diff
@@ -233,9 +234,11 @@ func (h *apiHandler) downloadArtifact(request *restful.Request, response *restfu
 		return
 	}
 	pipelineName := pr.Labels[v1alpha3.PipelineNameLabelKey]
+	isMultiBranch := pr.Spec.IsMultiBranchPipeline()
+	branchName := pr.GetRefName()
```

Comment:
Not sure if this is the right way to get branchName.

Reply:
Yes, it is right.

Reply:
thx